projects/crypto-currency/iluminados/api/src/routes/modules/allMarket.js

```diff
@@ -0,0 +1,30 @@
+import express from 'express'
```

Comment:
que tal chamar esse arquivo de markets?

Reply:
Eu vou criar um arquivo que contemple todas essas configurações e importa-los depois chamando em cada rota. Caso eu faça isso, irá ter um erro, pois nas demais rotas o serviço que eu estou chamando possui um nome diferente.
Exemplo: Coin of the Day